group.go

```diff
@@ -170,10 +178,67 @@ func (cli *Client) UpdateGroupParticipants(jid types.JID, participantChanges map
 	if err != nil {
 		return nil, err
 	}
-	// TODO proper return value?
 	return resp, nil
 }
 
+// GetGroupParticipantRequests get the list of participants waiting to join the specified group in WhatsApp.
+func (cli *Client) GetGroupParticipantRequests(jid types.JID) ([]*types.JID, error) {
+	res, err := cli.sendGroupIQ(context.TODO(), iqGet, jid, waBinary.Node{
+		Tag: "membership_approval_requests",
+	})
+	if err != nil {
+		return nil, err
+	}
+	participantsNode, ok := res.GetOptionalChildByTag("membership_approval_requests")
+	if !ok {
+		return nil, &ElementMissingError{Tag: "membership_approval_requests", In: "response to group participants requests"}
+	}
+	children := participantsNode.GetChildren()
+	participants := make([]*types.JID, 0, len(children))
+	for _, child := range children {
+		childAG := child.AttrGetter()
+		switch child.Tag {
+		case "membership_approval_request":
+			part := childAG.JID("jid")
+			if !childAG.OK() {
+				return nil, &ElementMissingError{Tag: "jid", In: "membership_approval_request"}
+			}
+			participants = append(participants, &part)
+		}
+	}
+	return participants, nil
+}
+
+// UpdateGroupParticipantRequests can be used to accept or reject requests to join WhatsApp groups.
+func (cli *Client) UpdateGroupParticipantRequests(jid types.JID, participantChanges map[types.JID]ParticipantChange) (*waBinary.Node, error) {
```

Comment:
The `membership_requests_action` as only accepting "approve" or "reject" changes, am i right?
if so, maybe we need to Add some validation around that?

or even create 2 separated functions, `ApproveGroupParticipantRequests` & `RejectGroupParticipantRequests`, so we can pass simple `[]JID` instead of map